go-controller/pkg/ovn/pods.go

```diff
@@ -176,6 +180,11 @@ func (oc *Controller) waitForNodeLogicalSwitch(nodeName string) (*net.IPNet, err
 	// The node switch will be created when the node's logical network infrastructure
 	// is created by the node watch.
 	var subnet *net.IPNet
+	hostSubnet, exists := oc.logicalSwitchCache[nodeName]
```

Comment:
@JacobTanenbaum still need to do this under the `oc.lsMutex`. just create a small function that you can call both here, and inside the PollImmediate that literally locks, defers unlock, reads the map, and returns the two values.